addons/rating/models/rating.py

```diff
@@ -35,7 +35,7 @@ def _default_access_token(self):
     def _selection_target_model(self):
         return [(model.model, model.name) for model in self.env['ir.model'].search([])]
 
-    create_date = fields.Datetime(string="Submitted on")
+    create_date = fields.Datetime(string="Submitted on", help="The date at which the rating is submitted to the client.")
```

Comment:
Not sure this help adds something ... it basically tells the same as the string, with a bit more words ^^ . Unless good reasons, I would avoid a new translation just for that.